docs/source/contribute.md

```diff
@@ -123,7 +123,7 @@ continue.
    requirements locally.
 
    ```bash
-   python3 -m pip install -e . -r dev-requirements.txt
+   python3 -m pip install -e ".[pycurl]" -r dev-requirements.txt
```

Comment:
I think maybe we don't need pycurl here, but I wasn't sure so I retained it to match current setup.

Reply:
I now know we need pycurl here and to run the test suite due to conftest.py imports MockAsyncHTTPClient which depends on a tornado Curl client that depends on pycurl.